plugins/vst_base/CMakeLists.txt

```diff
@@ -39,23 +39,43 @@ IF(LMMS_HOST_X86_64)
 	IF("${WINEBUILD_OUTPUT}" MATCHES ".*x86_64-linux-gnu/wine/libwinecrt0.a.*")
 		SET(EXTRA_FLAGS ${EXTRA_FLAGS} -nodefaultlibs /usr/lib/i386-linux-gnu/wine/libwinecrt0.a -luser32 -lkernel32 -lgdi32)
 	ENDIF()
-	#The following check works on Fedora systems 
-	IF("${WINEBUILD_OUTPUT}" MATCHES ".*lib64/wine/libwinecrt0.a.*")
+	# The following check works on Fedora systems
+	IF("${WINEBUILD_OUTPUT}" MATCHES "/usr/lib/lib64/wine/libwinecrt0.a.*")
 		SET(EXTRA_FLAGS ${EXTRA_FLAGS} -nodefaultlibs /usr/lib/i386/wine/libwinecrt0.a -luser32 -lkernel32 -lgdi32)
 	ENDIF()
+	# Development
+	IF("${WINEBUILD_OUTPUT}" MATCHES "/opt/wine-devel/lib64/wine/libwinecrt0.a.*")
+		SET(EXTRA_FLAGS ${EXTRA_FLAGS} -nodefaultlibs /opt/wine-devel/lib/wine/libwinecrt0.a -luser32 -lkernel32 -lgdi32)
+	ENDIF()
+	# Staging
+	IF("${WINEBUILD_OUTPUT}" MATCHES "/opt/wine-staging/lib64/wine/libwinecrt0.a.*")
+		SET(EXTRA_FLAGS ${EXTRA_FLAGS} -nodefaultlibs /opt/wine-staging/lib/wine/libwinecrt0.a -luser32 -lkernel32 -lgdi32)
+	ENDIF()
 ENDIF(LMMS_HOST_X86_64)
 
 SET(WINE_CXX_FLAGS "" CACHE STRING "Extra flags passed to wineg++")
 
 STRING(REPLACE " " ";" WINE_BUILD_FLAGS ${CMAKE_CXX_FLAGS} " " ${CMAKE_EXE_LINKER_FLAGS} " " ${WINE_CXX_FLAGS})
 ADD_CUSTOM_COMMAND(
-		SOURCE "${CMAKE_CURRENT_SOURCE_DIR}/RemoteVstPlugin.cpp"
-		COMMAND ${WINE_CXX}
-		ARGS "-I\"${CMAKE_BINARY_DIR}\"" "-I\"${CMAKE_SOURCE_DIR}/include\"" "-I\"${CMAKE_INSTALL_PREFIX}/include/wine/windows\"" "-I\"${CMAKE_INSTALL_PREFIX}/include\"" -I/usr/include/wine/windows "\"${CMAKE_CURRENT_SOURCE_DIR}/RemoteVstPlugin.cpp\"" -ansi -mwindows -lpthread ${EXTRA_FLAGS} -fno-omit-frame-pointer ${WINE_BUILD_FLAGS} -o ../RemoteVstPlugin
-		COMMAND sh -c "mv ../RemoteVstPlugin.exe ../RemoteVstPlugin || true"
-		TARGET vstbase
-		OUTPUTS ../RemoteVstPlugin
-		)
+	SOURCE "${CMAKE_CURRENT_SOURCE_DIR}/RemoteVstPlugin.cpp"
+	COMMAND ${WINE_CXX}
+	ARGS "-I\"${CMAKE_BINARY_DIR}\""
+		"-I\"${CMAKE_SOURCE_DIR}/include\""
+		"-I\"${CMAKE_INSTALL_PREFIX}/include/wine/windows\""
+		"-I\"${CMAKE_INSTALL_PREFIX}/include\""
+		-I/usr/include/wine/windows
+		-I/opt/wine-devel/include
+		-L/opt/wine-devel/lib
+		-I/opt/wine-staging/include
+		-I/opt/wine-staging/lib
```

Comment:
I've made a new changeset that detects if devel or stable is installed and includes directories of only one wine version.